src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ReleaseInternalNoteViewModel.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
+{
+    public class ReleaseInternalNoteViewModel
```

Comment:
I think ReleaseStatusViewModel would be a better name for this as it's something that exists for every status change and the note is just an attribute of it.